src/content/1.7/modules/concepts/templating/vuejs/_index.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Using VueJS
+menuTitle: VueJS
+weight: 2
+chapter: true
+---
+
+# Using VueJS
+
+Depending on the PrestaShop version running on the shop, either on the back or the front-office,
+at least one template renderer is available to display content on the page. With Javascript
+scripts, it is possible to update & animate the page content without having to refresh the page.
+
+jQuery is available on PrestaShop to help implementing these interface updates.
+For complex interfaces, with state management for instance, other frontend options could be considered.
+[VueJS](https://vuejs.org/) is a Javascript Framework that will be helpful in this case.
+This chapter covers its integration on a module.
+
+{{% children %}}
+
+## 1- Creation of a VueJS application in the module directories
+
+The objective of this part is to make VueJS available in the project. 
+
+We will store the VueJS source code in a dedicated folder that will be present in the module only
+during the development part. Running VueJS will be possible by compiling and storing the JS files
+in the proper assets folder.
+
+During this tutorial, we use a new folder `_dev` that will be created at the root directory of the module.
+
+Installation of VueJS can be done via [several ways](https://vuejs.org/v2/guide/installation.html),
+but we will use the CLI option to get all the stack.
+
+* Installation of VueJS CLI tool (globally)
+
+```
+$ npm install -g @vue/cli
+```
+
+* Creation of the project
+
+```
+$ vue create _dev
+```
+
+We chose the preset with Vue 2 & Babel for this tutorial.
```

Comment:
```suggestion
We chose the preset with Vue.js 2 & Babel for this tutorial.
```

Reply:
For this one I'd like to stick to the content displayed on the screen: They mention precisely 'Vue'

Reply:
@Quetzacoalt91 👍